packages/playwright/src/reporters/base.ts

```diff
@@ -419,13 +428,15 @@ export function formatResultFailure(screen: Screen, test: TestCase, result: Test
     });
   }
 
-  for (const error of result.errors) {
+  const errorPrompts = result.attachments.filter(a => a.name === 'errorPrompt');
+  result.errors.forEach((error, i) => {
     const formattedError = formatError(screen, error);
     errorDetails.push({
       message: indent(formattedError.message, initialIndent),
       location: formattedError.location,
+      prompt: errorPrompts[i]?.body?.toString('utf-8'),
```

Comment:
I think the assumption about those indexes matching is 1) too string 2) is actually not correct

Reply:
replaced it with error index in the attachment name